npm run install-tools does not work

I am completely new with all this frontend tools so I may have missed something.

The [documentation](https://github.com/acquia-pso/cog/blob/8.x-1.x/STARTERKIT/_readme/setup-full.md#local-development) mentions about `npm run install-tools` but it is failing.

> $ npm run install-tools
> npm ERR! Darwin 15.6.0
> npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "run" "install-tools"
> npm ERR! node v4.4.1
> npm ERR! npm  v2.14.20

> npm ERR! missing script: install-tools
> npm ERR! 
> npm ERR! If you need help, you may report this error at:
> npm ERR!     <https://github.com/npm/npm/issues>

I cant see any script or mention in any "executable" files.

Is the documentation outdated? Is this "install-tools" a default command from npm which is not working on my local?

--- a/.travis.yml
+++ b/.travis.yml
@@ -76,7 +76,7 @@ install:
 
   - ./install-node.sh 6.9.0
   - nvm use 6.9.0
-  - yarn install --force
+  - npm run install-tools
 
 # Run cog tasks.
 script:

--- a/STARTERKIT/package.json
+++ b/STARTERKIT/package.json
@@ -42,6 +42,7 @@
   "//": "The postinstall script is needed to work-around this Drupal core bug: https://www.drupal.org/node/2329453",
   "scripts": {
     "postinstall": "find node_modules/ -name '*.info' -type f -delete",
+    "install-tools": "yarn install --force",
     "uninstall-tools": "rm -r node_modules;",
     "build": "gulp",
     "build:dev": "gulp build:dev",

